test-e2e/specs/presenceSpec.js

```diff
@@ -0,0 +1,95 @@
+/* global describe, it, expect, jasmine */
+var DeepstreamServer = require( '../../../deepstream.io/src/deepstream.io' ),

```

Comment:
assumes folders are next to eachother, ie:
`- dev`
`-- deepstream.io`
`-- deepstream.io-client-js`